numpy/core/tests/test_regression.py

```diff
@@ -2412,3 +2412,27 @@ def test_field_access_by_title(self):
             v = str(data[['f1']])
             if HAS_REFCOUNT:
                 assert_(base <= sys.getrefcount(s))
+
+    @pytest.mark.skipif(sys.version_info < (3, 0), reason="Python3 only")
+    def test_datetime_memoryview(self):
```

Comment:
I think this test might be better alongside wherever the other memoryview tests are

Reply:
moved